doctoshotgun.py

```diff
@@ -763,6 +750,28 @@ def main(self, cli_args=None):
 
         vaccine_list = [docto.vaccine_motives[motive] for motive in motives]
 
+        
+
+        
+        class TimeandDate
```

Comment:
Proper CamelCase would be
```suggestion
        class TimeAndDate
```